front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -227,6 +229,11 @@ async function handler(
           isDataSourceAllowedInPlan = plan.limits.connections.isIntercomAllowed;
           assistantDefaultSelected = true;
           break;
+        case "microsoft_sharepoint":
+        case "microsoft_teams":
+          isDataSourceAllowedInPlan = true;
```

Comment:
why not follow the isXXXallowed pattern here?

Reply:
there's a lot of boilerplate for adding new limits in the plan, and was not required to start working. I'll create a separate task for that.